openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1340,6 +1340,7 @@ This specification defines the following Credential Issuer Metadata:
   * `enc_values_supported`: REQUIRED. Array containing a list of the JWE [@!RFC7516] encryption algorithms (`enc` values) [@!RFC7518] supported by the Credential and Batch Credential Endpoint to encode the Credential or Batch Credential Response in a JWT [@!RFC7519].
   * `encryption_required`: REQUIRED. Boolean value specifying whether the Credential Issuer requires the additional encryption on top of TLS for the Credential Response. If the value is `true`, the Credential Issuer requires the encryption for every Credential Response and therefore the Wallet MUST provide encryption keys in the Credential Request. If the value is `false`, the Wallet MAY chose whether it provides encryption keys or not.
 * `credential_identifiers_supported`: OPTIONAL. Boolean value specifying whether the Credential Issuer supports returning `credential_identifiers` parameter in the `authorization_details` Token Response parameter, with `true` indicating support. If omitted, the default value is `false`.
+* `signed_metadata`: OPTIONAL. String that is a signed JWT. This JWT contains Credential Issuer metadata parameters as claims. The signed metadata MUST be secured using JSON Web Signature (JWS) [@!RFC7515] and MUST contain an `iat` (Issued At) claim and an `iss` (Issuer) claim denoting the party attesting to the claims in the signed metadata. This JWT can also be used to identify the actual Credential Issuer when it is using third party service provider as a domain to host its metadata file, in which case, the JWT SHOULD contain `sub` (Subject) claim matching the Credentials Issuer identifier. If the Wallet supports signed metadata, metadata values conveyed in the signed JWT MUST take precedence over the corresponding values conveyed using plain JSON elements. A `signed_metadata` metadata value SHOULD NOT appear as a claim in the JWT. The Wallet MUST establish trust in the signer of the metadata, and obtain the keys to validate the signature. The concrete mechanism how to do that is out of scope of this specification and may be defined in the profiles of this specification.
```

Comment:
I read the proposal described in https://github.com/openid/OpenID4VCI/issues/80#issuecomment-1786501377 again and I noticed that the current proposal in this PR is different. The difference is that in https://github.com/openid/OpenID4VCI/issues/80#issuecomment-1786501377, the suggestion is to have the following JWT claim values in `signed_metadata`: 
- `iss` is actual credential issuer
- `sub` is the third-party service hosting the metadata aka `signed_metadata`

The proposal above ^ suggests that the credential issuer is attesting/issuing the JWT, whereas in this PR, it is the opposite as follows:
- `iss` is the third-party service hosting the metadata aka `signed_metadata`
- `sub` is the credential issuer identifier

I thought about the proposal in this PR and I cannot get my around it. If the `signed_metadata` is hosted under the domain of a third-party service provider which is different from the actual credential issuer's domain, how would you perform .well-known discovery? IMO, this won't be possible unless one has someone has for example a DNS redirect. If this assumption is wrong and the idea is to keep the same domain for credential issuer and third-party service provider, why would you need this feature at all?

Reply:
The current state of this PR reflects the discussion in this PR, so I don't think we should be hang up on the discussion in the issue 80. Bottomline, there is an entity that issues JWT (iss) and an entity that hosts the metadata (sub)  - how do you call each of those (actual issuer or the provider) does not matter as long as sub == credential issuer identifier

Reply:
Resolved by latest update of this PR (see comment on "actual" issuer).